docker/Dockerfile

```diff
@@ -18,7 +18,7 @@ COPY src ./src
 # FIXME: the data directory is specified in the project file and therefore gets
 # copied in a publish. This means that changes in the data changes trigger a
 # full rebuild.
-RUN dotnet publish -c release -o /app --no-restore
+RUN dotnet publish -c release -o /app --no-restore /restore
```

Comment:
What does the `/restore` do here?

Reply:
it gives this error if exclude it on my machine #22 1.185 /usr/share/dotnet/sdk/3.1.415/Sdks/Microsoft.NET.Sdk/targets/Microsoft.PackageDependencyResolution.targets(241,5): error MSB4018: The "ResolvePackageAssets" task failed unexpectedly. [/build/src/NadekoBot/NadekoBot.csproj]

Reply:
If you change it to `RUN dotnet publish -c release -o /app` does it work for you?

Reply:
yes that command works

Reply:
This means the Dockerfile and/or the dependencies are still broken somehow. The `/restore` is actually undoing the `--no-restore` cause `/flag` is the Windows way of adding command line flags to things and I guess they carried it over to the dotnet command.

Reply:
Are you saying the dockerfile fails to build on your machine or you need to run that command locally for it to work?

Reply:
i mean dockerfile fails to build